mne_bids/write.py

```diff
@@ -964,7 +964,7 @@ def make_dataset_description(path, name=None, data_license=None,
     _write_json(fname, description, overwrite=True, verbose=verbose)
 
 
-def write_raw_bids(raw, bids_basename, output_path, events_data=None,
+def write_raw_bids(raw, bids_basename, bids_root, events_data=None,
```

Comment:
are we willing to break this without deprecation cycle @agramfort ?

Reply:
I'd be willing to accept it without a deprecation cycle, given that it's a fairly obvious change and easy to adapt to. We can also make it prominent in the changelog.

Reply:
It's a good way to check if we really have users :-)